sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/ManagedDiskOperationsTests.java

```diff
@@ -317,7 +317,7 @@ public void canCopyStartIncrementalSnapshot() {
             .withCopyStart()
             .withIncremental(true)
             .create();
-        Assertions.assertFalse(snapshotSameRegion2.awaitCopyStartCompletion(Duration.ofMillis(1)));
+        Assertions.assertFalse(!isPlaybackMode() && snapshotSameRegion2.awaitCopyStartCompletion(Duration.ofMillis(1)));
```

Comment:
Disabled in playback because polling too fast and it actually succeeded?

@XiaofeiCao again be careful about timing in test.
Please also review change in SnapshotImpl

Reply:
Correct, only in playback though since the poll delay is set to 1ms meaning there is a bit of a race condition around this now.

Reply:
Right, in playback, 1ms is a long time.